VSRAD.Syntax/FunctionList/FunctionListProvider.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace VSRAD.Syntax.FunctionList
+{
+    public class FunctionListProvider
```

Comment:
Is this class needed?

Reply:
This is outdated code. You should take a look at the current implementation.